src/components/date.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+
+interface MyDateProps {
+    dateparam: string,
+    locals: string,
+    options: any
+  }
+
+const MyDate: FC<MyDateProps>= ({dateparam, locals = 'en-us', options = {timeZone: 'UTC', month:"long", day:"numeric", year:"numeric"}}) => {
```

Comment:
How about allowing the template writer to specify partial options? So, set `options = {}` in the function declaration, and set the full options in the body of the function:
```
const DEFAULT_OPTIONS = {timeZone: 'UTC', month:"long", day:"numeric", year:"numeric"};
const full_options = {...DEFAULT_OPTIONS, ...options};
```